### ISSUE TITLE
Please support /Za in <windows.h>

### ISSUE THREAD

**Issue by lovettchris** [microsoft/WindowsAppSDK#1390]
**Describe the bug**

I am stitching together some cross platform code and it very much needs to be compiled with /Za for lots of reasons.  But when I include <windows.h> from Windows SDK version 10.0.19041.0 I hit the following 4 bugs:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2467	illegal declaration of anonymous 'struct'	CppApp	C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winioctl.h	4337	
Error	C2467	illegal declaration of anonymous 'struct'	CppApp	C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winnt.h	12874	
Error	C2467	illegal declaration of anonymous 'struct'	CppApp	C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winnt.h	18837	
Error	C2133	'_IMAGE_POLICY_METADATA::Policies': unknown size	CppApp	C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winnt.h	21055	
```

**Steps to reproduce the bug**

1. Launch VS 2019
2. File New C++ console app
3. Add #include <windows.h> at the top
4. Go to C++ properties, C/C++, Language and "Disable Language Extensions" to Yes (/Za)
5. Build.

**Expected behavior**
Would be super cool if you could fix this, it would make building cross platform tools like Lean much easier.

**Screenshots**

**Version Info**

Windows SD 10.0.19041.0

NuGet package version: 

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (22000.176)              |    |

**Additional context**

**Comment by BenJKuhn** [comment]
I've confirmed that these issues still exist in the latest Windows preview SDK. We do generally strive to maintain standards compliance to some degree. I've opened OS Internal Windows work items to address these in a future release of the SDK. Thank you for reporting this issue. 

Unfortunately, there's no simple workaround at this time to enable /Za other than modifying the headers on disk to be conformant (which is a nuisance and is generally discouraged but can be done if no other options are palatable).

Thanks,

Ben Kuhn

For Microsoft tracking purposes, here are the work items IDs:
35909300
35909438
25909533
35909689
